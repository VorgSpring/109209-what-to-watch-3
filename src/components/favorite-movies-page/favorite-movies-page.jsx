import React from 'react';
import {Header} from '../header/header.jsx';
import {Footer} from '../footer/footer.jsx';
import {
  FavoriteMovieListContainer
} from '../favorite-movies-list/favorite-movies-list.jsx';

export const FavoriteMoviesPage = () => (
  <div className='user-page'>
    <Header
      title='My list'
      showUserBlok
      userType
    />

    <section className='catalog'>
      <h2 className='catalog__title visually-hidden'>Catalog</h2>

      <FavoriteMovieListContainer />
    </section>

    <Footer />
  </div>
);
