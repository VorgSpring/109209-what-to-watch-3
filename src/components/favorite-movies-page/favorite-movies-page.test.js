import React from 'react';
import {FavoriteMoviesPage} from './favorite-movies-page.jsx';
import renderer from 'react-test-renderer';

jest.mock(`../header/header.jsx`, () => ({
  Header() {
    return <main-header />;
  }
}));

jest.mock(`../favorite-movies-list/favorite-movies-list.jsx`, () => ({
  FavoriteMovieListContainer() {
    return <favorite-movies-list />;
  }
}));

jest.mock(`../footer/footer.jsx`, () => ({
  Footer() {
    return <main-footer />;
  }
}));

describe(`AddReviewPage`, () => {
  it(`should renders correctly`, () => {
    const tree = renderer.create(
        <FavoriteMoviesPage />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
