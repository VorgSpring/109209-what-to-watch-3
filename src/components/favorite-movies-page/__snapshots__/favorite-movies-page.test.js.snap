// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddReviewPage should renders correctly 1`] = `
<div
  className="user-page"
>
  <main-header />
  <section
    className="catalog"
  >
    <h2
      className="catalog__title visually-hidden"
    >
      Catalog
    </h2>
    <favorite-movies-list />
  </section>
  <main-footer />
</div>
`;
